react/components/molecules/chat-message/chat-message.js

```diff
@@ -57,9 +61,18 @@ export class ChatMessage extends PureComponent {
         return { marginTop: index > 0 ? 2 : 0 };
     };
 
+    _chatMessageComponent = () => {
+        return this.props.onPress ? Touchable : View;
+    };
+
     render() {
+        const ChatMessageComponent = this._chatMessageComponent();
         return (
-            <View style={[styles.chatMessage, this.props.style]}>
+            <ChatMessageComponent
+                style={[styles.chatMessage, this.props.style]}
+                underlayColor={this.props.underlayColor}
+                onPress={this.props.onPress}
+            >

```

Comment:
I think you will have to change the structure of the content inside this component (only one child), or else it throws this error in iOS:

<img width="403" alt="imagem" src="https://user-images.githubusercontent.com/25725586/161312875-e77c0ad9-41cd-4367-92a2-816973f3d179.png">